### DISCUSSION TITLE
Adding Pre-existing Weather Particles to Camera Views (Depth & Segmentation)

### DISCUSSION THREAD

**Discussion by Casey-J-Wolcott** [microsoft/AirSim#4512] [Support Q&A]
Hello,

I know this was mentioned before on discussion #3695 but was wondering if this change will be added to the snow and rain as well as part of the AirSimNH environment. I was able to successfully make these changes myself in the Blocks play environment in AirSim, but since AirSimNH has no editor I have no way to edit/replace the weather api files to enable the depth/segmentation camera views to see particles. 

I am working with this environment for a college project and it would be nice to have additional useable weather effects that actually have an effect on the image a drone is getting. I am not capable of building such a realistic environment myself that allows for my custom weather api files.

Thank you for your time.

**Comment by jonyMarino** [comment]
@Casey-J-Wolcott, you can build AirSim with a free environment from the UE marketplace like landscape mountains. Please take a look at our documentation [here](https://microsoft.github.io/AirSim/unreal_custenv/)

**Reply by Casey-J-Wolcott** [reply]
Thank you for the response.

I was hoping the weather would be properly fixed in all releases using Airsim. I have tried your link above, however there is no "learn" tab in the epic games launcher, and if I look under "samples" or "marketplace" I can't even find the example "Landscape Mountains". Perhaps the method of getting environments has changed since the doc was updated? This is all I have to work with.

**Reply by jonyMarino** [reply]
@Casey-J-Wolcott, sorry for the late response, [here](https://github.com/jonyMarino/LandscapeMountains), you can find the LandscapeMountains environment.
We will include the changes needed for the snow and rain in the next release, but I don't have an ETA for that.
